DataRepo/models.py

```diff
@@ -67,6 +67,33 @@ def atom_count_in_formula(formula, atom):
     return count
 
 
+class Protocol(models.Model):
+
+    MSRUN_PROTOCOL = "msrun_protocol"
+    ANIMAL_TREATMENT = "animal_treatment"
+    CATEGORY_CHOICES = [
+        (MSRUN_PROTOCOL, "LC-MS Run Protocol"),
+        (ANIMAL_TREATMENT, "Animal Treatment"),
+    ]
+
+    # Instance / model fields
+    id = models.AutoField(primary_key=True)
+    name = models.CharField(max_length=256, unique=True)
+    description = models.TextField(
+        blank=True, help_text="Full text of the protocol's methods"
+    )
+    category = models.CharField(
+        max_length=256,
+        choices=CATEGORY_CHOICES,
+        default=MSRUN_PROTOCOL,
+        help_text="Classification of the protocol, "
+        "referencing it's intended application",
```

Comment:
NB. In the current context of this review (having read the issue), this is clear, but thinking down the road, I can see this as being ambiguous (and only elucidated by looking at the values in the database).  I suggest something like:
```suggestion
        help_text="Type of protocol, e.g. an animal treatment or MSRun lab "
        "procedure",
```

Reply:
I added the specific examples, but did not use "Type" because in fan's proposal, there was a separate `type` attribute, that might still be called into play in the (unknown) future.